### PR TITLE
[main] upgrading OpenIPMI to v2.0.32

### PR DESCRIPTION
**Pull request by MaxBrodeurUrbas**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

What does the PR accomplish, why was it needed?

###### Change Log  

- upgrading OpenIPMI to v2.0.32

###### Does this affect the toolchain?  

**NO**

###### Test Methodology

- Local build and install on test vm

### CODE REVIEW

File: SPECS/OpenIPMI/OpenIPMI.spec [hunk 1]
```diff
@@ -1,17 +1,15 @@
 Summary:        A shared library implementation of IPMI and the basic tools
 Name:           OpenIPMI
-Version:        2.0.25
-Release:        7%{?dist}
+Version:        2.0.32
```
Comment (mfrw) [inline SPECS/OpenIPMI/OpenIPMI.spec:3]: I think we missed to update the `%changelog`

File: cgmanifest.json [hunk 2]
```diff
@@ -15584,8 +15584,8 @@
         "type": "other",
         "other": {
           "name": "OpenIPMI",
-          "version": "2.0.25",
-          "downloadUrl": "https://sourceforge.net/projects/openipmi/files/latest/download/openipmi-2.0.25.tar.gz"
+          "version": "2.0.32",
+          "downloadUrl": "http://downloads.sourceforge.net/openipmi/OpenIPMI-2.0.32.tar.gz"
```
Comment (mfrw) [inline cgmanifest.json:15588]: NIT: lets convert `s/http/https/g`
```suggestion
          "downloadUrl": "https://downloads.sourceforge.net/openipmi/OpenIPMI-2.0.32.tar.gz"
```

File: SPECS/OpenIPMI/OpenIPMI.spec [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
+Release:        1%{?dist}
 License:        LGPLv2+ AND GPLv2+ OR BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          System Environment/Base
 URL:            https://sourceforge.net/projects/openipmi/
-Source0:        https://sourceforge.net/projects/openipmi/files/latest/download/openipmi-%{version}.tar.gz
+Source0:        http://downloads.sourceforge.net/openipmi/OpenIPMI-2.0.32.tar.gz
```
Comment (mfrw) [inline SPECS/OpenIPMI/OpenIPMI.spec:10]:
```suggestion
Source0:        https://downloads.sourceforge.net/openipmi/OpenIPMI-2.0.32.tar.gz
```

File: SPECS/OpenIPMI/OpenIPMI.spec [hunk 4]
```diff
@@ -1,17 +1,15 @@
...
 Source1:        openipmi-helper
 Source2:        ipmi.service
-# Enable detection of python versions with deprecated distutils modules (Source: Fedora 28, MIT license)
-Patch0:         %{name}-py39.patch
```
Comment (mfrw) [inline SPECS/OpenIPMI/OpenIPMI.spec:14]: We might have missed to remove the checked in `patch` file :)

**Review by mfrw — CHANGES_REQUESTED** [review]
For some reasons when i do a local build (not a `mock` build), the build fails with a warning and an error:
```
File listed twice: /usr/share/licenses/OpenIPMI/COPYING"
File not found: /usr/src/mariner/BUILDROOT/OpenIPMI-2.0.32-1.cm2.x86_64/usr/lib/perl5/vendor_perl"
```

Although, via `mock` the build works. I would request if we can double check.

**Comment by MaxBrodeurUrbas** [comment]
> For some reasons when i do a local build (not a `mock` build), the build fails with a warning and an error:
> 
> ```
> File listed twice: /usr/share/licenses/OpenIPMI/COPYING"
> File not found: /usr/src/mariner/BUILDROOT/OpenIPMI-2.0.32-1.cm2.x86_64/usr/lib/perl5/vendor_perl"
> ```
> 
> Although, via `mock` the build works. I would request if we can double check.

Thanks for catching some of my mistakes, went a bit too quick on this one :/ However I have built it locally without mock and installed all the rpms on a test 2.0 VM. I just rebuilt it to be sure now. Are you able to reproduce that build issue? Should I run a pipeline build to be sure?

**Review by mfrw — APPROVED** [review]
LGTM.
Given you have ensured it builds.
Thank you for the cleanup as well :)

**Pull request merged**